wurst/extensions/UnitExtensions.wurst

```diff
@@ -245,3 +243,28 @@ public function unit.isBoss() returns bool
 // TODO: Use UnitEnterLeave so that this is called automatically.
 public function unit.correctColor()
     this.setColor(this.getOwner().getColor())
+
+/**
+    This was made solely to disable quick make ability
+    TODO: Handle multiple requirement string
+*/
+public function unit.disableAbility(int abilId, string requirement)
+    let abil = this.getAbility(abilId)
+    let tooltipNormExt = abil.getField(ABILITY_SLF_TOOLTIP_NORMAL_EXTENDED, 0)
+    let requirementString = (
+            "Requires :\n" +
+            "    " + requirement + "\n\n").color(COLOR_YELLOW)
+    abil.setField(ABILITY_SLF_TOOLTIP_NORMAL_EXTENDED, 0, requirementString + tooltipNormExt)
+    this.disableAbility(abilId, true, false)
+
+/**
+    This was made solely to enable quick make ability
+*/
+public function unit.enableAbility(int abilId)
+    let abil = this.getAbility(abilId)
+    // Fun fact, this method takes an abilId as parameter, it actually update all abilities with this id
```

Comment:
What does this comment mean? Why would `getAbilityExtendedTooltip` update any ability, let alone all abilities with the same ID?

Reply:
Hmm I can't remember why I wrote this comment, I know I wanted to precise the difference between updating an ability globally and updating an ability just for an unit, but this feels out of place.